src/Apps/IdentityVerification/routes.tsx

```diff
@@ -3,6 +3,10 @@ import { RouteConfig } from "found"
 import { graphql } from "react-relay"
 
 export const routes: RouteConfig[] = [
+  {
+    path: "/identity-verification/processing",
+    getComponent: () => loadable(() => import("./Processing")),
```

Comment:
For this route (and all below) we should be preloading the split bundle like so:

Assign to a variable:
https://github.com/artsy/reaction/blob/master/src/Apps/Artwork/routes.tsx#L4

Load behind `prepare`: 
https://github.com/artsy/reaction/blob/master/src/Apps/Artwork/routes.tsx#L9-L12